### PR TITLE
Update CoreDNS manifest

### PR DESCRIPTION
**Pull request by rajansandeep**
Update the manifest for CoreDNS to keep up with the latest changes.

### CODE REVIEW

File: deploy/addons/coredns/coreDNS-svc.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-    kubernetes.io/name: "CoreDNS"
+    k8s-app: kube-dns
     addonmanager.kubernetes.io/mode: Reconcile
+    kubernetes.io/name: "CoreDNS"
 spec:
   selector:
-    k8s-app: coredns
-  clusterIP: 10.0.0.10
+    k8s-app: kube-dns
+  clusterIP: 10.96.0.10
```
Comment (johnbelamaric) [inline deploy/addons/coredns/coreDNS-svc.yaml]: This needs to match kubelet, are you sure we want to change it?

Comment (rajansandeep) [reply deploy/addons/coredns/coreDNS-svc.yaml]: It's changed for kube-dns as well. Keeping in sync.

Comment (r2d4) [reply deploy/addons/coredns/coreDNS-svc.yaml]: This was changed in https://github.com/kubernetes/minikube/pull/2077

Sorry for missing it in this addon!

File: deploy/addons/coredns/coreDNS-svc.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: coredns
+  name: kube-dns
```
Comment (luxas) [inline deploy/addons/coredns/coreDNS-svc.yaml]: FWIW; you might wanna share the code between the kube-dns & CoreDNS Services as they should be kinda equal
not very important point though...

**Comment by dlorenc** [comment]
@minikube-bot OK to test

**Pull request merged**